packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -55,9 +55,10 @@ Future<void> buildWindows(WindowsProject windowsProject, BuildInfo buildInfo, {
 
   // TODO(pbo-linaro): Add support for windows-arm64 platform, https://github.com/flutter/flutter/issues/129807
   const TargetPlatform targetPlatform = TargetPlatform.windows_x64;
-  final Directory buildDirectory = globals.fs.directory(
-    getWindowsBuildDirectory(targetPlatform)
-  );
+  final Directory buildDirectory = globals.fs.directory(globals.fs.path.join(
+    projectPath,
```

Comment:
@loic-sharma do you think this is safe? essentially changing this from a relative path to an absolute one?